### PR TITLE
feat(#33): gh actions for check broken/missing links

### PR DESCRIPTION
**Pull request by ervinismu**

### CODE REVIEW

File: .github/workflows/checklinks.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - uses: actions/checkout@v3
+      - name: Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 3.1
+      - name: Setup Rubygems, Bundler, jekyll
+        run: |
+          gem update --system --no-document
+          gem update bundler --no-document
+          gem install jekyll bundler
```
Comment (postmodern) [inline .github/workflows/checklinks.yaml:25]: Is `bundler` getting installed twice here? I'm pretty sure we can use the default version of `bundler`.

File: .github/workflows/checklinks.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+      - name: Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 3.1
+      - name: Setup Rubygems, Bundler, jekyll
+        run: |
+          gem update --system --no-document
```
Comment (postmodern) [inline .github/workflows/checklinks.yaml:23]: This might be overkill. Whatever version of rubygems that shipped with the installed ruby is probably fine.

**Review by postmodern — CHANGES_REQUESTED** [review]
Think we can remove `gem update --system` and `gem update bundler`. `bundler` should be installed by default and shouldn't need updating (yet).

**Comment by ervinismu** [comment]
updated @postmodern , sorry for late response

**Pull request merged**